modules/core/portal-bootstrap/system.packages.extra.bnd

```diff
@@ -297,5 +296,6 @@ Provide-Capability:\
 	@lib/com.liferay.petra.sql.dsl.spi.jar,\
 	@lib/com.liferay.petra.string.jar,\
 	@lib/com.liferay.petra.url.pattern.mapper.jar,\
-	@lib/com.liferay.petra.xml.jar
+	@lib/com.liferay.petra.xml.jar,\
+	@lib/com.liferay.portal.log4j.jar
```

Comment:
@julschong 
1. You should not change petra.xml
2. Why log4j jar needed to be included? Can we remove it?

Reply:
@tinatian Hi Tina,
I tried removing but error occurs when I started the server

Reply:
@julschong What kind of error?

Reply:
@julschong I removed and see an error about portal-log4j-extender resolving, it is unable to find "com.liferay.portal.log4j" when it is deployed as the portal-log4j module had not been started at that moment. I will let this pull go and let's have a talk about this tomorrow.

Tina

Reply:
Sorry for late response, I was rerunning this to get the error log as well.
OK, I will rebase this and we talk about it tomorrow